### ISSUE TITLE
[Bug]: Problem on Startup after Installation

### ISSUE THREAD

**Issue by phexerus** [homebridge-eufy-security/plugin#150]
### What happened?

Hi Team,
After the installation, no popup showed up, and i get this error in the status console of my homebridge
Can you help me out?
Best regards 
Christian

[20/08/2022, 00:13:35] [Homebridge UI] [homebridge-eufy-security] /usr/local/lib/node_modules/homebridge-eufy-security/homebridge-ui/server.js:107
      this.config['country'] = body.country ??= 'US';
                                            ^^^

SyntaxError: Unexpected token '??='
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47

### Device Type

other (UnSupported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v14 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
[20/08/2022, 00:13:35] [Homebridge UI] [homebridge-eufy-security] /usr/local/lib/node_modules/homebridge-eufy-security/homebridge-ui/server.js:107
      this.config['country'] = body.country ??= 'US';
                                            ^^^

SyntaxError: Unexpected token '??='
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47
```

**Comment by phexerus** [comment]
Yes, the error is not inside the common issues :(

**Comment by thieren** [comment]
Hi @phexerus 

This is a duplicate of #58

You can update to 2.1.0-rc. It is fixed there, or you update your node version to v14 or v16.
Since newer Homebridge versions do not support node v12 anymore I'd suggest updating both the plugin (to release candidate and your node version.

**Comment by thieren** [comment]
Is fixed with release of 2.1.0
